plan/property/task_type.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package property
+
+// TaskType is the type of execution task.
+type TaskType int
+
+const (
```

Comment:
Should the `TaskType` be in the `property`'s package?

Reply:
the present planner treats the task type as an physical property, so put it here.